### PR TITLE
Delete unused code

### PR DESCRIPTION
**Pull request by deletescape**
Do not merge yet, just here so it can be reviewed in case I delete stuff which would still be needed

### CODE REVIEW

**Comment by deletescape** [comment]
@timusus can com.simplecity.amp_library.glide.utils.LoggingListener be deleted or will this be used for something in the future?

**Comment by timusus** [comment]
Can't find your comment, but happy for you to remove the Glide `LoggingListener`

**Comment by deletescape** [comment]
Work should be done for now, you can merge it if the changes are alright @timusus

File: app/src/main/java/com/simplecity/amp_library/playback/MultiPlayer.java [hunk 1]
```diff
@@ -184,17 +183,7 @@ void setVolume(float vol) {
...
-            try {
-                sessionId = mCurrentMediaPlayer.getAudioSessionId();
-            } catch (IllegalStateException ignored) {
-                //Nothing to do
-            }
-            return sessionId;
-        } else {
-            return 0;
-        }
+        return mCurrentMediaPlayer.getAudioSessionId();
```
Comment (timusus) [inline app/src/main/java/com/simplecity/amp_library/playback/MultiPlayer.java]: The try-catch should remain in-tact here. It's not related to the SDK version - there's a chance the `MediaPlayer` is in a state where `getAudioSessionId()` will throw an `IllegalStateException`.

**Pull request merged**